test/fixtures/render/mixin-override.json

```diff
@@ -221,7 +221,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"tasks\": [\n    {\n      \"status\": \"CREATED\",\n      \"name\": \"one\"\n    },\n    {\n      \"status\": \"CREATED\",\n      \"name\": \"two\"\n    }\n  ]\n}"
+                              "content": "{\n  \"tasks\": [\n    {\n      \"status\": \"\",\n      \"name\": \"one\"\n    },\n    {\n      \"status\": \"\",\n      \"name\": \"two\"\n    }\n  ]\n}"
```

Comment:
I think this is a regression. Previously the following enum produce the rendered value `CREATED` whereas it is now an empty string.

```apib
## Status
+ status (enum[string])
    + `CREATED`
    + `QUEUED`
    + `RUNNING`
    + `FINISHED`
    + `ERROR
```